libs/navigation/website/src/lib/navbar/navbar.tsx

```diff
@@ -145,106 +155,155 @@ export const NavPaperWrapper = styled('div')`
   left: 0;
   right: 0;
   transform: translateY(100%);
+
+  ${({theme}) => css`
+    ${theme.breakpoints.up('md')} {
+      gap: ${theme.spacing(6)};
+      grid-row-gap: ${theme.spacing(12)};
+      grid-template-columns: 1fr 1fr;
+      padding: ${theme.spacing(2.5)} calc(100% / 6) calc(100% / 12);
+    }
+  `}
 `
 
-export const NavPaperCategoryList = styled('div')`
+export const NavPaperCategory = styled('div')`
   display: grid;
-  gap: ${({theme}) => theme.spacing(3)};
+  gap: ${({theme}) => theme.spacing(1)};
+  grid-auto-rows: max-content;
+`
+
+export const Name = styled('span')`
+  text-transform: uppercase;
+  font-weight: 300;
+  font-size: 14px;
```

Comment:
No hardcoding of font-sizes ideally! We have predefined font sizes that can be changed. For example the whole thing could be `body2` if those styles are reoccuring